test/unit/payments/get-payments-by-scheme-id.test.js

```diff
@@ -0,0 +1,26 @@
+jest.mock('../../../app/payments/get-data')
+const { getData: mockGetData } = require('../../../app/payments/get-data')
+
+const { SCHEME_ID: SCHEME_ID_CATEGORY } = require('../../../app/constants/categories')
+const { SCHEME_ID_VALUE } = require('../../../app/constants/scheme-id-value')
+
+const { DATA } = require('../../mocks/values/data')
+
+const { getPaymentsBySchemeId } = require('../../../app/payments')
+
+describe('get payments by frn', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+    mockGetData.mockResolvedValue(DATA)
```

Comment:
Space needed here

Reply:
where sorry? im getting a linting error when putting a space between the `describe` and `beforeEach`

Reply:
Between jest.clearAllMocks and mockGetData.mockResolvedValue